README.md

```diff
@@ -45,6 +49,8 @@ The list of props for the AsyncApi React component includes:
 
      > **NOTE:** The `Partial<T>` type means that every field in the `T` type is optional.
 
+   > **NOTE:** You must pass `schema` or `urlSchema` to component for correct work of component.
```

Comment:
What about rewriting this copy?

Either `schema` or `urlSchema` prop must be present.